e2e_playwright/st_arrow_vega_lite_chart_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+def test_arrow_vega_lite_chart(themed_app: Page):
```

Comment:
nit: since this isn't really testing any theming-related aspects, it could also just use `app` to only run the test once:

```suggestion
def test_arrow_vega_lite_chart(app: Page):
```